src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -830,4 +830,52 @@ Do you want to proceed?</value>
   <data name="Paste_Tracking" xml:space="preserve">
     <value>Paste Tracking</value>
   </data>
+  <data name="Invalid_assembly_name" xml:space="preserve">
+    <value>Invalid assembly name</value>
+  </data>
+  <data name="Invalid_characters_in_assembly_name" xml:space="preserve">
+    <value>Invalid characters in assembly name</value>
+  </data>
```

Comment:
these two are quite unexpected to see.

Reply:
Good eye. They were missing from a previous PR. Once I rebase/merge it should go away.